content/docs/command-reference/plots/modify.md

```diff
@@ -1,10 +1,11 @@
 # plots modify
 
-Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+Modify display properties of
+[stage plots](/doc/command-reference/plots#stage-plots) files.
 
 > ⚠️ Note that this command can modify only data-series plots. It has no effect
-> on image-type plots. See
-> [Types of metrics](/doc/command-reference/plots#types-of-metrics).
+> on image-type plots and standalone plots. See
+> [Types of plots](/doc/command-reference/plots#description).
```

Comment:
Users can click previous links to find out about general info on plots, no need to update this link since that section is gone (we can remove it). Committing
```suggestion
> on image-type plots and standalone plots.
```